main_test.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"testing"
+	"net/http"
+	realZipper "github.com/go-graphite/carbonzipper/zipper"
+	"go.uber.org/zap"
+	"net/http/httptest"
+	"github.com/stretchr/testify/assert"
+	pb "github.com/go-graphite/carbonzipper/carbonzipperpb3"
+	"github.com/go-graphite/carbonapi/expr"
+	"context"
+	"github.com/lomik/zapwriter"
+	"encoding/json"
+)
+
+type MockCarbonZipper struct {
```

Comment:
not sure that it's worth to make it Public at this moment. Otherwise it should be moved to a separate package.